pkg/logger/logger.go

```diff
@@ -251,10 +253,12 @@ func (l *Logger) GetFileLineString(calldepth int) string {
 }
 
 // GetClient parses an HTTP request for the client/remote IP address.
-func GetClient(req *http.Request) string {
-	client := req.Header.Get("X-Real-IP")
-	if client == "" {
-		client = req.RemoteAddr
+func GetClient(reverseProxy bool, req *http.Request) string {
```

Comment:
I think it's "best practice" to have `req` as the first arg when a func deals with a HTTP request.
```suggestion
func GetClient(req *http.Request, reverseProxy bool) string {
```